### PR TITLE
Added included and third-party plugin documentation

### PR DESCRIPTION
**Pull request by jmwright**
Added listings with descriptions for both included and third-party plugins.

### CODE REVIEW

File: third_party.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Third-Party Plugins
+
+This is a curated list of CadQuery plugins that are hosted outside of this repository. The core development team of CadQuery does not control these repositories, so extra care must be taken by the user to make sure these plugins are safe to use.
+
+## Contributing
+
+If you are interesting in having a plugin included in this list, it must meet the following criteria.
+
+* Must have a README file with a description of the plugin, installation instructions, and an example of how to use the plugin.
+* Must be able to pass a code review for safety. This is not a check to make sure the plugin is bug-free.
```
Comment (dcowden) [inline third_party.md:10]: I propose that we require third party plugins to conform to the rules for installation as those in the contrib.  Though this applies some constraints, it also ensures that the overall community benefits from a standard. 

I would propose further that we offer to support other standards, if they emerge and are reasonable. I would prefer one plugin format. I could live with a couple, if there's a good reason. I think > 3 is detrimental.
If a contributor proposes any alternate formats that seem reasonable, we could consider having o

Comment (jmwright) [reply third_party.md:10]: I think part of your message got cut off here.

Comment (dcowden) [reply third_party.md:10]: oops sorry! edited, i had re-phrased and that was just a leftover

Comment (jmwright) [reply third_party.md:10]: The third bullet point [here](https://github.com/CadQuery/cadquery-plugins/blob/676ebb69e3375fff8f2606cd6f131c98d97f8171/third_party.md) addresses this I think.

**Comment by jmwright** [comment]
Will close #7 and #9

**Pull request merged**